includes/abstracts/abstract.llms.post.model.php

```diff
@@ -1188,24 +1206,131 @@ public function toArray() {
 
 		ksort( $arr ); // Because i'm anal...
 
-		return apply_filters( 'llms_' . $this->model_post_type . '_to_array', $arr, $this );
+		/**
+		 * Filter the final post array created when converting the object to an array
+		 *
+		 * The dynamic portion of this hook, `$this->model_post_type`, refers to the model's post type. For example "course"m
+		 * "lesson", "membership", etc...
+		 *
+		 * @since [version]
+		 *
+		 * @param array           $arr   Associative array of the model.
+		 * @param LLMS_Post_Model $model Post model instance.
+		 */
+		return apply_filters( "llms_{$this->model_post_type}_to_array", $arr, $this );
 
 	}
 
 	/**
 	 * Called before data is sorted and returned by $this->toArray()
+	 *
 	 * Extending classes should override this data if custom data should
-	 * be added when object is converted to an array or json
+	 * be added when object is converted to an array or json.
 	 *
-	 * @param    array $arr   array of data to be serialized
-	 * @return   array
-	 * @since    3.3.0
-	 * @version  3.3.0
+	 * @since 3.3.0
+	 *
+	 * @param array $arr Array of data to be serialized.
+	 * @return array
 	 */
 	protected function toArrayAfter( $arr ) {
 		return $arr;
 	}
 
+	/**
+	 * Add "extra" data to the post array during export/serialization
+	 *
+	 * This method adds two arrays of data, "blocks" and "images".
+	 *
+	 * The "blocks" array is an array of reusable blocks used in the post's content. During
+	 * an import these blocks will be imported into the site.
+	 *
+	 * The "images" array is an array of image element source URLs found in the post's content. During
+	 * an import these images will be imported into the new site via media sideloading.
+	 *
+	 * @since [version]
+	 *
+	 * @param array $arr Post array from `toArray()`.
+	 * @return array[]
+	 */
+	protected function to_array_extra( $arr ) {
+
+		$arr['_extra'] = array(
```

Comment:
```suggestion
		$arr['_extras'] = array(
```
the generators look for '_extras' not for '_extra'